openstack/destroy.sh

```diff
@@ -1,17 +1,12 @@
 #!/bin/bash
 set -euo pipefail
 
-# SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
-# source "$SCRIPT_DIR/common.sh"
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
+echo "Deleting server"
 
-# echo "Backing up history file into: '$STATION_HISTORY_BACKUP'"
-# scp -r "$EIRINI_STATION_USERNAME@$STATION_IP:~/.zsh_history" "$STATION_HISTORY_BACKUP" || true
+openstack server delete "$EIRINI_STATION_USERNAME-eirini-station" --wait
```

Comment:
In the `opensetack/provision.sh` script we restore the history from the local copy, so we should be saving a local copy of the shell history prior to deleting the server here.

Reply:
added the back-up back-in